docs/crocks/Identity.md

```diff
@@ -0,0 +1,61 @@
+# Identity
+
+`Identity a`
+
+Crock which returns the same value that was used as its argument
+
+## Why?
+
+To expose the constructor and instances defined below
+
+| Constructor | Instance |
+|:---|:---|
+| [`of`](#of) | [`ap`](#ap), [`chain`](#chain), [`equals`](#equals), [`map`](#map), [`of`](#of), [`sequence`](#sequence), [`traverse`](#traverse), [`value`](#value) |
+
+## Constructors
+
+### of
+
+`Identity a => a -> Identity a`
+
+## Instances
+
+### inspect
+
+`Identity a => Identity a`
+
+### value
+
+`() -> a`
+
+### type
+
+`() -> Identity`
+
+### equals
+
+`a -> Boolean`
+
+### map
+
+`Identity m => m (a -> b) -> m b`
+
+### ap
+
+`Identity m => m a ~> m (a -> b) -> m b`
```

Comment:
This siggy matches the Fantasy Land spec and it just so happens this is the only place in `crocks` that does not match the spec (for reasons). So to get it to show how `crocks` handles `Apply`, it should be something like:
`Identity m => m (a -> b) ~> m a -> m b`.

Notice how it is reversed.